mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,404 @@
+            ch_name = _read_str(fid, 10)
+            ch_names.append(ch_name)
+            fid.seek(data_offset + 75 * ch_idx + 4)
+            if np.fromfile(fid, dtype='u1', count=1)[0]:
+                bads.append(ch_name)
+            fid.seek(data_offset + 75 * ch_idx + 19)
+            pos.append(np.fromfile(fid, dtype='f4', count=2))  # x and y pos
+            fid.seek(data_offset + 75 * ch_idx + 47)
+            # Baselines are subtracted before scaling the data.
+            baselines.append(np.fromfile(fid, dtype='i2', count=1)[0])
+            fid.seek(data_offset + 75 * ch_idx + 59)
+            sensitivity = np.fromfile(fid, dtype='f4', count=1)[0]
+            fid.seek(data_offset + 75 * ch_idx + 71)
+            cal = np.fromfile(fid, dtype='f4', count=1)
+            cals.append(cal * sensitivity * 1e-6 / 204.8)
+
+        fid.seek(event_offset)
+        event_type = np.fromfile(fid, dtype='<i1', count=1)[0]
+        event_size = np.fromfile(fid, dtype='<i4', count=1)[0]
+        if event_type == 1:
+            event_bytes = 8
+        elif event_type == 2:
+            event_bytes = 19
+        else:
+            raise IOError('Unexpected event size.')
+
+        n_events = event_size // event_bytes
+        stim_channel = np.zeros(n_samples)  # Construct stim channel
+        for i in range(n_events):
+            fid.seek(event_offset + 9 + i * event_bytes)
+            event_id = np.fromfile(fid, dtype='u2', count=1)[0]
+            fid.seek(event_offset + 9 + i * event_bytes + 4)
+            offset = np.fromfile(fid, dtype='<i4', count=1)[0]
+            event_time = (offset - 900 - 75 * n_channels) // (n_channels * 2)
+            stim_channel[event_time - 1] = event_id
+
+    info = _empty_info(sfreq)
+    if lowpass_toggle == 1:
+        info['lowpass'] = highcutoff
+    if highpass_toggle == 1:
+        info['highpass'] = lowcutoff
+    subject_info = {'hand': hand, 'id': patient_id, 'sex': sex,
+                    'first_name': first_name, 'last_name': last_name}
+
+    if eog == 'auto':
+        eog = [ch for ch in ch_names if 'EOG' in ch.upper()]
+    if ecg == 'auto':
+        ecg = [ch for ch in ch_names if 'ECG' in ch.upper()]
+    if emg == 'auto':
+        emg = [ch for ch in ch_names if 'EMG' in ch.upper()]
+    eegs = list()
+    for idx, ch_name in enumerate(ch_names):
+        if ch_name in eog or idx in eog:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_EOG_CH
+        elif ch_name in ecg or idx in ecg:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_ECG_CH
+        elif ch_name in emg or idx in emg:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_EMG_CH
+        elif ch_name in misc or idx in misc:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_MISC_CH
+        else:
+            coil_type = FIFF.FIFFV_COIL_EEG
+            kind = FIFF.FIFFV_EEG_CH
+            eegs.append(idx)
```

Comment:
same lines here: https://github.com/mne-tools/mne-python/blob/master/mne/io/nicolet/nicolet.py#L120 and maybe here too: https://github.com/mne-tools/mne-python/blob/master/mne/realtime/fieldtrip_client.py#L158 and other places.